frontend/src/views/camp/Dashboard.vue

```diff
@@ -0,0 +1,149 @@
+                  <template v-for="day in period.days().items">
+                    <template v-if="showDay(day)">
+                      <v-col :key="day._meta.self" cols="12" class="pb-0">
+                        <h4>{{ day.dayOffset + 1 }}) {{ displayDate(day) }}</h4>
+                      </v-col>
+                      <template v-for="scheduleEntry in day.scheduleEntries().items">
+                        <v-col
+                          v-if="showScheduleEntry(scheduleEntry)"
+                          :key="scheduleEntry._meta.self"
+                          :set="scheduleEntryTime = defineHelpers(scheduleEntry)"
+                          cols="12" sm="6"
+                          md="4" lg="3">
+                          <v-card
+                            :color="scheduleEntry.activity().category().color"
+                            :to="scheduleEntryRoute(camp(), scheduleEntry)">
+                            <v-card-title>
+                              {{ scheduleEntry.activity().category().short }} {{ scheduleEntry.number }}: {{ scheduleEntry.activity().title }}
+                              <v-spacer />
+                              <user-avatar
+                                v-for="cc in sortCampCollaborations(scheduleEntry.activity().campCollaborations().items)"
+                                :key="cc._meta.self"
+                                :camp-collaboration="cc" :size="24"
+                                style="margin: 2px" />
+                            </v-card-title>
+                            <v-card-subtitle>
+                              {{ $date.utc(scheduleEntryTime.startTime).format($tc('global.datetime.dateShort')) == $date.utc(scheduleEntryTime.endTime).format($tc('global.datetime.dateShort'))
+                                ? '' : $date.utc(scheduleEntryTime.startTime).format($tc('global.datetime.dateShort')) }}
+                              <b> {{ $date.utc(scheduleEntryTime.startTime).format($tc('global.datetime.hourShort')) }} </b>
+                              -
+                              {{ $date.utc(scheduleEntryTime.startTime).format($tc('global.datetime.dateShort')) == $date.utc(scheduleEntryTime.endTime).format($tc('global.datetime.dateShort'))
+                                ? '' : $date.utc(scheduleEntryTime.endTime).format($tc('global.datetime.dateShort')) }}
+                              <b> {{ $date.utc(scheduleEntryTime.endTime).format($tc('global.datetime.hourShort')) }} </b>
+                            </v-card-subtitle>
+                          </v-card>
+                        </v-col>
+                      </template>
+                    </template>
+                  </template>
+                </v-row>
+              </v-container>
+            </v-expansion-panel-content>
+          </v-expansion-panel>
+        </template>
+      </v-expansion-panels>
+    </template>
+    <template v-else>
+      <v-card-text>
+        <p style="text-align: center">
```

Comment:
This style should be defined in css, especially since it is repeated in line 78.
Maybe there is a component for that?

Reply:
Vuetify has a helper class: `class="text-center"`
This is better than inline style in case we ever want to introduce a CSP policy.